integrationTests/tsconfig.json

```diff
@@ -1,6 +1,9 @@
 {
   "extends": "../tsconfig.json",
   "compilerOptions": {
+    "target": "ES2020",
```

Comment:
Is this relevant?

Reply:
that was a rebase conflicts not properly solved (duplicate values) - should disappear now from the changed files